subsys/bluetooth/audio/CMakeLists.txt

```diff
@@ -38,9 +38,9 @@ zephyr_library_sources_ifdef(CONFIG_BT_MCC mcc.c)
 
 zephyr_library_sources_ifdef(CONFIG_BT_MCS mcs.c)
 
-zephyr_library_sources_ifdef(CONFIG_BT_MCS mpl.c)
+zephyr_library_sources_ifdef(CONFIG_BT_MPL mpl.c)
 
-zephyr_library_sources_ifdef(CONFIG_BT_MCS media_proxy.c)
+zephyr_library_sources_ifdef(CONFIG_MCTL media_proxy.c)
```

Comment:
I am quite unsure about the mapping between media proxy and media control. While they are very similar, it just reads weird that control enables a proxy. Suggest to unify the names.

Reply:
I have been thinking along the same lines.  (Does that mean that we are great minds? :-) )
I have been thinking that the media_proxy could be renamed to media_control.  (That would be a separate PR, I think.)
Opinions?

Reply:
I actually like "media control" more than "media proxy" :)

Reply:
Please create a GH issue to rename it if we want to do that, or rename the Kconfig to match "proxy"

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/43107